deepdrivemd/misc/OutlierDB.py

```diff
@@ -1,9 +1,10 @@
 import os
 import random
+from typing import List, Tuple  # noqa
```

Comment:
Remove noqa

Reply:
It looks like Tuple is not used, that can be removed from the import.

Reply:
Done